docker-compose.yml

```diff
@@ -107,10 +120,15 @@ services:
       context: ./client-report
       args:
         GIT_HASH: "${GIT_HASH}"
+    volumes:
+      - "config:/app/config"
 
 networks:
   polis-dev:
 
 volumes:
   backups:
   postgres:
+  config:
```

Comment:
Fwiw, using `./config/` instead of a _named volume_ like `config`, is a special case, and not the same implementation under the covers. It seems that mounting with `./config` (instead of a named volume) aggravates certain edge-cases which we're probably best to stay away from...? Basically, we can lean on a docker container to copy in config if we use anonymous volumes, plus it muddles `docker-machine` stuff

E.g., https://stackoverflow.com/a/45728328/504018

Reply:
And naming the volume explicitly (instead of letting docker-compose autogenerate a namespace) allowed for more easily scripting stuff relating to volumes. E.g., if your local project was in `~/repos/polisnew` and mine in `~/mystuff/polisServer`, then my volume would be named `polisnew_config` and yours would be `polisserver_config`. That's no good if we want to write a script that modifies contents of volume directly